src/pages/public_registration/PublicProjectsContent.tsx

```diff
@@ -32,14 +45,23 @@ interface PublicProjectsContentProps {
 }
 
 export const PublicProjectsContent = ({ projects }: PublicProjectsContentProps) => {
-  const { t } = useTranslation('registration');
+  const { t } = useTranslation('project');
 
   return (
     <>
       <Typography variant="h4" component="h2" gutterBottom>
         {t('registration:description.project_association')}
       </Typography>
 
+      <StyledHeadingRow>
+        <Typography variant="caption">{t('common:title')}</Typography>
+        <span />
+        <Typography variant="caption">{t('common:institution')}</Typography>
+        <span />
+        <Typography variant="caption">{t('project_manager')}</Typography>
+        <span />
+        <Typography variant="caption">{t('project_info')}</Typography>
+      </StyledHeadingRow>
       <MuiThemeProvider theme={lightTheme}>
```

Comment:
så dere bruker theme på enkeltkomponenter?

Reply:
Bare i nød. Når vi har mørkeblå bakgrunn bruker vi egen `darkTheme` som setter skriftfarge til  hvit (siden svart ikke virker på mørkeblå). Her er det en hvit rad for hvert prosjekt, så da må vi flett inn `lightTheme` igjen slik at det blir svart skrift etc igjen.

Ideelt sett kunne vi definert alt dette i `darkTheme`, feks i darkTheme skal skrift være hvit, men for Chip så skal den være svart fordi Chip er lys. Men det ble fort mye knot, så vi har bare brukt denne enkle fiksen i stedet